examples/sugarscape_g1mt/app.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import solara
+from matplotlib.figure import Figure
+from mesa_models.experimental import JupyterViz
+from sugarscape_g1mt.model import SugarscapeG1mt
+from sugarscape_g1mt.resource_agents import Sugar
+from sugarscape_g1mt.trader_agents import Trader
+
+
+def space_drawer(viz):
```

Comment:
@tpike3 what do you think of this custom space drawer? I use `imshow` because doing `ax.scatter()` on 2x50x50 resources is very slow. This would have been much harder to write with old Mesa frontend framework, which needs to be written in Python and JS.

Reply:
That is awesome! Did you have to worry about the origin with imshow? If I remember correctly it is in the upper left vs lower left, as the mesa default?

Reply:
Did you have to invert when using `imshow` in the complexity tutorial? The i, j convention of `imshow` here is the same as the `scatter`.

Reply:
> Did you have to invert when using `imshow` in the complexity tutorial? The i, j convention of `imshow` here is the same as the `scatter`.

Yes for the tutorial I had to do ---- plt.imshow(resource_distribution, origin="lower")

Reply:
Should I do that too in this PR? It's hard to check after each steps if the resource being eaten corresponds to the agents' location.